hw/hw_0001/index.2.js

```diff
@@ -1,21 +1,28 @@
-let goods = [
-    {name: "Beer", cost: 1.5},
-    {name: "Milk", cost: 2.5},
-    {name: "Bread", cost: 4.5},
-    {name: "Butter", cost: 1.5},
-    {name: "Sausages", cost: 3},
+let products = [
+  {name: "Beer", number: "0,5L", price: 17.90},
```

Comment:
rename number -> weight
add field quantity with the available amount of product

Reply:
Ok, 
- renamed `number`
- add `quantity`
- add `sale`

Reply:
> * add `sale`
Please, changes its value from boolean to number according to the updated description